src/planscape/rscripts/forsys.R

```diff
@@ -154,20 +158,29 @@ preprocess_metrics <- function(metrics, condition_name) {
   return(metrics)
 }
 
+get_metric_column <- function(condition_name) {
+  column <- METRIC_COLUMNS[[condition_name]]
+  if (is.null(column)) {
```

Comment:
style nit/question: I think maybe R throws a warning if we try to access the element of a list without checking whether it exists first? Would something like `exists(condition_name, METRIC_COLUMNS)` avoid that?